friends/src/components/AddFriend.js

```diff
@@ -1,24 +1,65 @@
-import React from 'react'
-import styled from 'styled-components'
+import React from "react";
+import styled from "styled-components";
+import axios from "axios";
 
-const Form = styled.form`
-    margin: 25px 0 50px 0;
-`
-const FormHeader = styled.h1`
-    margin: 50px 0 0 0;
-`
-const AddFriend = (props) => {
+class AddFriend extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      name: "",
+      age: "",
+      email: ""
+    };
+  }
+
+  inputHandler = event => {
+    this.setState({ [event.target.name]: event.target.value });
+  };
+
+  submitHandler = event => {
+    event.preventDefault();
+    const newFriend = {
+      name: this.state.name,
+      age: parseInt(this.state.age, 10),
+      email: this.state.email
+    };
+    axios.post("http://localhost:5000/friends", newFriend).then(response => {
```

Comment:
Great job on your progress so far Keegan! You can also try storing your url in a variable to make things easier to type. 👍 Keep it up!